src/SDKs/Batch/DataPlane/Azure.Batch/Azure.Batch.csproj

```diff
@@ -28,7 +28,7 @@
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
-    <PackageReleaseNotes>For detailed release notes, see: https://aka.ms/batch-net-dataplane-changelog</PackageReleaseNotes>
+    <PackageReleaseNotes>For detailed release notes, see: https://aka.ms/batch-net-dataplane-changelog.</PackageReleaseNotes>
```

Comment:
minor: Remove the dot? (it'd be easy to copy that into the URL since nuget doesn't do links currently)